classes/item.rb

```diff
@@ -13,4 +13,17 @@ def initialize(publish_date)
     @publish_date = publish_date
     @archived = false
   end
+
+  def move_to_archive?
+    @archived = true if can_be_archived?
+  end
+
+  private
+
+  def can_be_archived?
+    now = Date.today
+    difference_in_years = ((now - @publish_date)).to_i
+    should_be_archived = (difference_in_years / 365).to_i
+    should_be_archived >= 10
```

Comment:
please change the logic of the ```should_be_archived``` variable to look like this:
if (should_be_archived > 10) ? true : false 
 as stated in the task link below

[task Link ](https://github.com/aimalamiri/Ruby-Catalog/issues/3)

Reply:
Thanks, @mwenyoa for indicating the issue.

I finished this.

Reply:
Addressed ✔